storage/iam.js

```diff
@@ -47,7 +47,7 @@ function viewBucketIamMembers (bucketName) {
       });
     })
     .catch((err) => {
-      console.error('Error:', err);
+      console.error('ERROR:', err);
```

Comment:
Every time I've added this catch statement, I've done `ERROR`. Let's get together tomorrow and pound out a Node.js sample style guide.